src/FakeItEasy.dotnet/FakeItEasy.dotnet.nuget.targets

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(NuGetPackageRoot)' == ''">
+    <NuGetPackageRoot>$(UserProfile)\.nuget\packages\</NuGetPackageRoot>
+  </PropertyGroup>
+  <ImportGroup>
+    <Import Project="$(NuGetPackageRoot)\StyleCop.MSBuild\4.7.54\build\StyleCop.MSBuild.Targets" Condition="Exists('$(NuGetPackageRoot)\StyleCop.MSBuild\4.7.54\build\StyleCop.MSBuild.Targets')" />
+  </ImportGroup>
+</Project>
```

Comment:
The dotnet build removes this file, so maybe "add and don't ignore" isn't the straightforward solution we'd hoped.

Specifically, the `.nuget\nuget.exe restore .\FakeItEasy-NetCore.sln -PackagesDirectory .\packages` whacks it.